apis/builder/blinded_blocks.yaml

```
@@ -14,6 +14,8 @@ post:
     After Deneb, this endpoint will additionally return the associated blobs in the response.
 
     After Fulu, this endpoint will be deprecated.
+    
+    After Fulu, this endpoint will not support JSON requests and responses. Client will have to send an SSZ encoded request and receive an SSZ encoded response.
```

Can simplify a bit. We should wrap these lines too.
```suggestion
    After Fulu, this endpoint will not support JSON requests and responses. Clients MUST send an SSZ encoded request and receive an SSZ encoded response.
```

---

This same comment applies to other endpoints.